styles.css

```diff
@@ -149,3 +149,585 @@ li:hover {
 ul {
   list-style-type: none;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
```

Comment:
`The main container of the projects was supposed to have a display:grid` to align the work cards. The purpose of this suggestion was to design the mobile design in a way that when we will be going to design the desktop version, it would be seamless for the alignment of the work cards.